### ISSUE TITLE
No links available when running with Spring Boot 1.4.0.RELEASE

### ISSUE THREAD

**Issue by thomasdarimont** [olivergierke/spring-restbucks#64]
No links are shown for the root resource in the hal-browser 
when running spring-restbucks with Spring Boot 1.4.0.RELEASE.

Wanted to use this as an example for real REST APIs with Spring Boot 1.4.0.RELEASE.
Got it working (with a quick hack) by only redirecting requests to the `hal-browser` that are not requests for `application/hal+json` via `headers={"!Accept=*application/hal+json*"}`.
in `org.springframework.data.rest.webmvc.halbrowser.HalBrowser`.
```java
	@RequestMapping(value = { "/", "" }, method = RequestMethod.GET, produces = MediaType.TEXT_HTML_VALUE, headers={"!Accept=*application/hal+json*"})
	public View index(HttpServletRequest request) {
		return getRedirectView(request, false);
	}
```

**Comment by olivergierke** [comment]
I think what you're seeing is [DATAREST-863](https://jira.spring.io/browse/DATAREST-863). That has been fixed already and the version of Spring Data REST is included in Spring Boot 1.4.1, so an upgrade to that should fix the issue.

**Comment by olivergierke** [comment]
I've committed the upgrade to Boot 1.4.1 against this ticket and the browser seems to work as expected again.
